circle.yml

```diff
@@ -395,11 +395,6 @@ jobs:
       - checkout
       - attach_workspace:
           at: ~/build
-      - spectest:
-          skip_validation: true
-          expected_passed: 10467
-          expected_failed: 3
-          expected_skipped: 1343
       - spectest:
           expected_passed: 11409
```

Comment:
At least set default values for these so we don't need to update them in 3 places.